Tests/UnitTests/OrderTests.swift

```diff
@@ -106,28 +105,69 @@ class OrderTests: XCTestCase {
         let id = "derp123"
         let amount: Int64 = 499
         let lineItems: [Order.LineItem] = []
+        let customerDictionary: [String: AnyHashable] = [:]
         let order = Order(id: id, amount: amount)
         let date = order.purchaseDate
         let expectedOrderDictionary: [String: AnyHashable] = ["order_id": id,
                                                               "amount": amount,
                                                               "currency": "USD",
                                                               "purchase_date": date,
-                                                              "line_items": lineItems ]
+                                                              "line_items": lineItems,
+                                                              "customer": customerDictionary]
 
         // Act
         guard let actualOrderDictionary = order.dictionaryRepresentation as? [String: AnyHashable] else {
             XCTFail("malformed dictionary")
             return
         }
 
-        print()
-        print("expected: \(expectedOrderDictionary)")
-        print()
-        print("actual: \(actualOrderDictionary)")
-        print()
-
         // Assert
         XCTAssertEqual(expectedOrderDictionary, actualOrderDictionary)
     }
+    
+    func testCustomerInitialization_hashingEmail() {
+        let id = "123"
+        let email = "betty@usebutton.com"
+        let emailSha256 = email.lowercased().sha256
```

Comment:
May want to use the actual hashed value so we aren't relying on another implementation of ours:

`C399E8D0E89E9F09AA14A36392E4CB0D058AB28B16247E80EAB78EA5541A20D3`